fl4health/losses/mkmmd_loss.py

```diff
@@ -1,6 +1,3 @@
-# multi-kernel maximum mean discrepancy
-# cao bin, HKUST, China, binjacobcao@gmail.com
-# free to charge for academic communication
```

Comment:
We departed from this a fair bit ago...so I think we can remove the attribution 😂